docs/source/guides/libraries.rst

```diff
@@ -67,12 +67,256 @@ Python library
 Where to get
 ^^^^^^^^^^^^
 
+Prerequirements:
+
+- cmake(3.11 or higher)
+- git
+- g++
+- boost(1.65 or higher, only system and filesystem)
+- swig(3.0.12 can be built --without-pcre)
+- protobuf
+
+Install iroha python libraries:
+
+- Through pip
+
+    .. code:: sh
+
+      pip install iroha
+
+    For the latest version
```

Comment:
I am not sure if this is exactly latest. What is our process for pip update? Do we update testpypi with version in development, and pypi with master?

Reply:
Yes